### PR TITLE
B js webpack config plugin

### PR DESCRIPTION
**Pull request by degiorgioo**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [ ] The commit message follows our guidelines: link tbd.
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] tooling / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
Actually no plugin for loading js-files is included within the webpack-config-plugins. 

Issue Number: Issue [#7](https://github.com/namics/webpack-config-plugins/issues/7)

## What is the new behavior?
Short: Create `js-config-webpack-plugin` which loads js-files. Files are loaded with the babel-loader.

Versions:

| Babel        |Babel-loader   |  
|--------------|--------------|
| 7.0.0-rc.1  | 8.0.0-beta.4 | 

## Does this PR introduce a breaking change?

```
[x] Yes
[ ] No
```
Described in the `new behaviour`-section.

### CODE REVIEW

File: packages/js-config-webpack-plugin/config/production.config.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
```
Comment (jantimon) [inline packages/js-config-webpack-plugin/config/production.config.js]: please remove this line

File: packages/js-config-webpack-plugin/test/fixtures/simple/.specific.babelrc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (jantimon) [inline packages/js-config-webpack-plugin/test/fixtures/simple/.specific.babelrc]: maybe we shouldn’t put a .specific.babel.rc inside the simple fixture case but rather in a custom babel rc case - what do you think?

File: packages/js-config-webpack-plugin/package.json [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        "npm": ">=5"
+    },
+    "scripts": {
+        "test": "jest --runInBand --verbose --forceExit",
+        "posttest": "tsc",
+        "test-watch": "jest --watch --verbose --forceExit",
+        "update-snapshots": "jest -u --runInBand"
+    },
+    "peerDependencies": {
+        "typescript": ">=2.0.0",
```
Comment (jantimon) [inline packages/js-config-webpack-plugin/package.json]: why do we need typescript here?

File: packages/js-config-webpack-plugin/package.json [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        "jsdom": "^11.12.0",
+        "rimraf": "2.6.2",
+        "sourcemap-validator": "1.1.0",
+        "typescript": "3.0.1",
+        "webpack": "^4.16.5"
+    },
+    "dependencies": {
+        "babel-loader": "^8.0.0-beta.4",
+        "thread-loader": "^1.2.0",
+        "@babel/core": "^7.0.0-rc.1",
```
Comment (jantimon) [inline packages/js-config-webpack-plugin/package.json]: please move babel core and babel env to dev dependencies as they are already peer dependencies

File: packages/js-config-webpack-plugin/test/fixtures/simple/.specific.babelrc (hunk 2 above)
Comment (degiorgioo) [reply packages/js-config-webpack-plugin/test/fixtures/simple/.specific.babelrc]: Ok. It sounds good. I will create a new folder within the testing directory which is base of the babel rc case.

File: packages/js-config-webpack-plugin/package.json (hunk 3 above)
Comment (degiorgioo) [reply packages/js-config-webpack-plugin/package.json]: Is not needed. It was a copy-paste error.

**Comment by janbiasi** [comment]
@degiorgioo please fill-out the issue description :) thanks in advance!

**Comment by jantimon** [comment]
Should we also add the react preset to be on the same level as the ts-loader?

https://babeljs.io/docs/en/babel-preset-react

**Pull request merged**